README.rst

```diff
@@ -257,10 +257,14 @@ default value:
   Determine whether pdb++ starts in sticky mode or not.
 
 ``line_number_color = Color.turquoise``
-  The color to use for line numbers.
+  The color to use for line numbers. 
+  
+  (Note: Color.* are taken from fancycompleter so remember to import this module first.)
 
 ``filename_color = Color.yellow``
   The color to use for file names when printing the stack entries.
+  
+  (Note: Color.* are taken from fancycompleter so remember to import this module first.)
```

Comment:
Thanks.  Makes sense in general I think.
Note however that you can also use strings there directly, and even more (see `current_line_color = "39;49;7"` below (but have not really tested it with `line_number_color` / `filename_color`.

Reply:
So the hint could be:
(Note: Color.* are taken from fancycompleter so remember to import this module first
or resort to direct formatting like "39;49;7".)